webapp/app.py

```diff
@@ -9,11 +9,10 @@
 from .users import login_manager
 from .sockets import socketio
 from .static_optimizer import cache_buster, compress
-
+from .users import app
 # to temporarily disable non-crucial pylint errors in conformity
 # pylint: disable=invalid-name,missing-docstring,no-value-for-parameter
 
-app = Flask(__name__)
```

Comment:
There's no need to write a `init_app()` function because [flask_sqlalchemy.SQLAlchemy already has one](https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#configuration), so I agree. Change it to `db = SQLAlchemy()`, put the `app` variable (along with all the moved `login_manager.init_app(app)` line) back into the `app.py` file and import db into `app.py`. Then you can use `db.init_app(app)` in `app.py`. The `app.config["key"] = value` stuff should also live in `app.py`. This may seem trivial, but it is essential to keep orgaization of the code conformitive, otherwise we'll be looking at yet another refactoring crusade over the winter break (not productive).